python/model.py

```diff
@@ -321,15 +330,39 @@
 
         # Making the variables
         X = keras.preprocessing.sequence.pad_sequences(features,
-                                                       maxlen=225,
                                                        padding='post')
 
+        # DAN Model feeds in all features at once, so there's no need to limit to the
```

Comment:
@scotthlee I mentioned it here, but the existing code padded to a hardcoded 225 (sequence length for the LSTM), but since the DAN is just flattening all the features into a single "bag", we can/should probably just pad to the largest bag size and then take that as the input shape.